DuckDuckGo/ContentBlockerViewController.swift

```diff
@@ -27,8 +27,8 @@ class ContentBlockerViewController: UITableViewController {
     @IBOutlet weak var httpsLabel: UILabel!
     @IBOutlet weak var blockCountCircle: UIImageView!
     @IBOutlet weak var blockCount: UILabel!
-    @IBOutlet weak var blockingEnabledToggle: UISwitch!
     @IBOutlet weak var blockThisDomainToggle: UISwitch!
+    @IBOutlet weak var tutorialText: UILabel!
```

Comment:
There's a bit of duplication here and in the `ContentBlockerDisabledViewController`: a couple of lines of code in each view controller, but also some UI in the story board.  Could extract the tutorial in to a view controller of its own and embed it on each of the popups instead.